src/reducers/versions.tsx

```diff
@@ -757,12 +792,19 @@ const reducer: Reducer<VersionsState, ActionType<typeof actions>> = (
     case getType(actions.toggleExpandedPath): {
       const { path, versionId } = action.payload;
 
-      const { expandedPaths } = state.versionInfo[versionId];
+      const version = state.versionInfo[versionId];
+
+      if (!version) {
+        _log.error(`Version missing for versionId: ${versionId}`);
```

Comment:
I wonder if this should throw an actual error, rather than just logging. It shouldn't really be possible to have called `toggleExpandedPath` without having a corresponding version loaded.